components/date-picker/lib/popups/date-range-popup.component.ts

```diff
@@ -98,9 +102,76 @@ export class DateRangePopupComponent implements OnInit, OnChanges {
     }
   }
 
+  ngAfterViewInit(): void {
+    (this.calendarBox.nativeElement as HTMLElement).focus(); // Focus on the calendar panel to avoid focus on the <input> by default
+  }
+
+  ngOnDestroy(): void {
+    this.keyDownSubscription.unsubscribe();
+  }
+
+  onKeyDown(event: KeyboardEvent): void {
+    if (event.target && (event.target as HTMLElement).nodeName.toLowerCase() === 'input') {
+      return ;
+    }
+
+    const ctrlKey = event.ctrlKey || event.metaKey; // NOTE: metaKey is the "⌘ Command" key on `MAC` and the "⊞ Windows" key on `Windows`
+    switch (event.keyCode) {
```

Comment:
Maybe it is a good idea that  the `event.preventDefault()` should be outside the switch block